### ISSUE TITLE
FR: Smaller width for pen

### ISSUE THREAD

**Issue by BryanHogan** [zsviczian/obsidian-excalidraw-plugin#2149]
### Is your feature request related to a problem? Please describe.

Hey, all 4 options currently available for the pen widths are far too fat to write. 

### Describe the solution you'd like

Would be great if there were a smaller width available that works well for writing text. Maybe some way to set the width freely would also work.

### Real-life usage scenarios

Being able to write text using the pen on my tablet.

### Describe alternatives you've considered

Zooming out and setting font size to something larger. Creating custom pens, but smaller width are not selectable even within the custom pens settings.

**Comment by green3554** [comment]
> use the flat tip custom pen: https://youtu.be/P_Q6avJGoWI?si=nbbDWyYBnskqncEZ&t=808

This isn't really a solution because the fine tip pin has an annoying taper, and trying to remove the taper results in a random blob at the end of each stroke. I would like to see stroke width customization in increments of 0.05 such as other pen properties.

**Comment by BryanHogan** [comment]
Yes. I also found these other pens to be lacking in one way or another, at least to achieve what I wanted, which is just a smaller pen width. I was able to create a pen that was smaller than the default one, but couldn't get the width down to what I wanted. I tried different configurations for a whole day but couldn't achieve that, some option to the copy main pen and then control its width would have been great. So I still would love to see some finer control in the options to adjust the width of the main pen, or some other solution.
Currently in the settings the stroke width is locked to steps of 0.5, maybe it would be possible to change this to 0.05, 0.1 or similar.

**Comment by arximboldi** [comment]
This is particularly true when using an external tablet (e.g. Wacom). The pen width is ridiculously large compared to the width of boxes and other diagram elements.

**Comment by zsviczian** [comment]
Consolidating pen-width and handwriting ergonomics discussion into #2775.

Closing this as a duplicate so related requests stay centralized. Please continue discussion there.
